gateway/server.go

```diff
@@ -113,10 +113,18 @@ func main() {
 	}
 
 	r := mux.NewRouter()
+	// max wait time to start a function = maxPollCount * functionPollInterval
 
+	scalingConfig := handlers.ScalingConfig{
+		MaxPollCount:         uint(1000),
+		FunctionPollInterval: time.Millisecond * 10,
+		CacheExpiry:          time.Second * 5, // freshness of replica values before going stale
+	}
+
+	scalingProxy := handlers.MakeScalingHandler(faasHandlers.Proxy, queryFunction, scalingConfig)
```

Comment:
I think this should be feature flagged before releasing - i.e. env-var: `scale_from_zero: true` - we might also need to load the "ScalingConfig" via env-vars at some point in the future. Let's see how useful the defaults are first.